go.mod

```diff
@@ -4,6 +4,6 @@ go 1.12
 
 require (
 	github.com/FortAwesome/Font-Awesome v0.0.0-20220831210243-d3a7818c253f // indirect
```

Comment:
This is not the latest version of Font-Awesome, this line should read:

```
	github.com/FortAwesome/Font-Awesome v0.0.0-20221115183454-96cafbd73ec4 // indirect
```

Use this command for upgrade:

```
hugo mod get -u github.com/FortAwesome/Font-Awesome@96cafbd73ec4
```

Reply:
Resolved with commit 3031c33b5e0b.